tests/modules/individuals/resources/test_featured_asset_guid.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+from tests.modules.sightings.resources import utils as sighting_utils
+from tests.modules.asset_groups.resources import utils as asset_group_utils
+from tests.modules.individuals.resources import utils as individual_utils
+from tests.modules.annotations.resources import utils as annot_utils
+from tests import utils
+
+
+def test_patch_featured_asset_guid_on_individual(db, flask_app_client, researcher_1):
+
+    # this test is a monster because it involves almost all of the major modules
+
+    from app.modules.sightings.models import Sighting
+
+    data_in = {
+        'encounters': [{}],
+        'startTime': '2000-01-01T01:01:01Z',
+        'locationId': 'test',
+    }
+
+    response = sighting_utils.create_sighting(
+        flask_app_client, researcher_1, data_in=data_in
+    )
+
+    from app.modules.encounters.models import Encounter
+
+    response_json = response.json
+
+    import logging
+
+    log = logging.getLogger(__name__)
+    log.warning('RETURNED ENC JSON: ' + str(response_json))
+
+    assert response_json['result']['encounters']
```

Comment:
I don't think this is doing what you think it is, I think it should be 
assert 'encounters' in response_json['result'] 
and you also need an 
assert 'resullt' in response.json 
first